packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -12,6 +12,11 @@ import {
 } from '@agoric/zoe/src/contractSupport/index.js';
 import { makeNotifierKit } from '@agoric/notifier';
 
+import {
+  CHARGING_PERIOD,
+  RECORDING_PERIOD,
+} from '@agoric/zoe/src/contracts/loan';
+
```

Comment:
That's an example contract that, according to #1656, should move out of Zoe. Let's not use that as the source of truth for this.